lib/i18n/backend/active_record.rb

```diff
@@ -51,6 +51,10 @@ def lookup(locale, key, scope = [], options = {})
           key = normalize_flat_keys(locale, key, scope, options[:separator])
           result = Translation.locale(locale).lookup(key)
 
+          if result.empty? && key == '.'
```

Comment:
Why don't we call this before or instead `result = Translation.locale(locale).lookup(key)`
Maybe smth like
```ruby
result = if key == '.'
  Translation.locale(locale).all
else
  Translation.locale(locale).lookup(key)
end
```
This way we won't run two queries for `.` key